src/main/pg/max7456.h

```diff
@@ -19,10 +19,6 @@
 
 #include "pg/pg.h"
 
-#define MAX7456_CLOCK_CONFIG_HALF 0
-#define MAX7456_CLOCK_CONFIG_OC   1
-#define MAX7456_CLOCK_CONFIG_FULL 2
-
```

Comment:
Actually I think this belongs here, since it is a `#define` just for the config value.

Reply:
Agreed that it belongs there. I have changed this with a view of the future plan to generate all `pg/*.h` files programmatically. For this it will be beneficial if these files only contain the 'naked' parameter group declarations, and all dependencies are added through includes. What are your thoughts on this?

Reply:
Well, these values represent valid values for the PG settings, so should be specified in the generator file and generated. Similarly for min and max values for settings.

But in any case I don't think we should anticipate it. That is should be thinking "I'll probably need to move these when we start generating the PGs, so I'll move them now". Rather we should move them if and when the PG generator requires it.

Reply:
Fair enough, withdrawing this.